test/unittests/test_excel2xml.py

```diff
@@ -275,9 +274,9 @@ def test_make_date_prop(self) -> None:
     def test_make_decimal_prop(self) -> None:
         prop = "decimal"
         method = excel2xml.make_decimal_prop
-        different_values = ["3.14159", 3.14159, .1, 100.0, "100.0"]
-        invalid_values = ["100", ".1", 100]
-        run_test(self, prop, method, different_values, invalid_values)
+        different_values = ["3.14159", 3.14159, "1.3e3", "100", ".1", 100]
+        invalid_values = ["string"]
+        run_test(self, prop, method, [float(x) for x in different_values], invalid_values)
```

Comment:
This is because I improved validation. It makes much more sense to let Python do the work for me, and to just accept everything that Python accepts. But until now, the tests were tailored to my own (old) validation. So I had to adapt the tests